bots/images/scripts/rhel.setup

```diff
@@ -91,6 +103,18 @@ name=appstream
 baseurl=http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/$REPO/compose/AppStream/x86_64/os/
 enabled=1
 gpgcheck=0
+
+[RHEL-8-NIGHTLY-BaseOS-Debug]
+name=baseos-debug
+baseurl=http://download-ipv4.eng.brq.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.1/compose/BaseOS/x86_64/debug/tree/
```

Comment:
Please use s/latest-RHEL-8.1/$REPO/ here, like in the other two stanzas. Stopping "bots" label for this to avoid unnecessary rebuilds.

Reply:
Done